tests/payment.yml

```diff
@@ -69,3 +77,7 @@ GatewayMessage:
     ClassName: AuthorizedResponse
     Reference: authorizedPaymentReceipt
     PaymentID: =>Payment.payment6
+  message6:
```

Comment:
Should have it's own number, currently there are two entries named `message6`. This throws off the other tests.